packages/foam-vscode/src/features/wikilink-reference-generation.ts

```diff
@@ -135,10 +135,10 @@ async function generateReferenceList(
 ): Promise<string[]> {
   const filePath = doc.fileName;
 
-  const id = dropExtension(basename(filePath));
```

Comment:
wow nice catch! I totally missed this in my code! Odd that TS didn't notice the type mismatch

Reply:
I found this on stackoverflow: https://stackoverflow.com/questions/51577959/is-it-possible-to-type-check-string-aliases-in-typescript

Reply:
ahh got it!